monitor/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mavromat/bgpstream-redis:v1.0
```

Comment:
we only need bgpstream on this image and now can remove it from backend

Reply:
perfect!